templates/blink/intent_to_implement.html

```diff
@@ -35,21 +54,17 @@ <h4>Contact emails</h4>
     {% endfor %}
   {% endif %}
 
+  {% if feature.search_tags %}
+    <br><br><h4>Search tags</h4>
+    {{feature.search_tags |join:", "}}
+  {% endif %}
+
```

Comment:
I'm curious why we'd include search tags in the intents mail.

Reply:
Should we link to a query in chromestatus?

Reply:
I made them links to queries.  I am thinking that the value of including this field is so that people can find related features for comparison.